src/app/features/ledger/hooks/use-ledger-navigate.ts

```diff
@@ -59,55 +63,101 @@ export function useLedgerNavigate() {
+          replace: true,
+          state: {
+            chain,
+            backgroundLocation: { pathname: RouteUrls.Home },
+          },
+        });
       },
 
       toErrorStep(chain: SupportedBlockchains, errorMessage?: string) {
         return navigate(RouteUrls.ConnectLedgerError, {
           replace: true,
-          state: { latestLedgerError: errorMessage, chain },
+          state: {
+            latestLedgerError: errorMessage,
+            chain,
+            backgroundLocation: { pathname: RouteUrls.Home },
+          },
         });
       },
 
       toAwaitingDeviceOperation({ hasApprovedOperation }: { hasApprovedOperation: boolean }) {
         return navigate(RouteUrls.AwaitingDeviceUserAction, {
           replace: true,
-          state: { hasApprovedOperation },
+          state: {
+            hasApprovedOperation,
+            backgroundLocation: { pathname: RouteUrls.Home },
+          },
         });
       },
 
       toPublicKeyMismatchStep() {
-        return navigate(RouteUrls.LedgerPublicKeyMismatch, { replace: true });
+        return navigate(RouteUrls.LedgerPublicKeyMismatch, {
+          replace: true,
+          state: {
+            backgroundLocation: { pathname: RouteUrls.Home },
+          },
+        });
       },
 
       toDevicePayloadInvalid() {
-        return navigate(RouteUrls.LedgerDevicePayloadInvalid, { replace: true });
+        return navigate(RouteUrls.LedgerDevicePayloadInvalid, {
+          replace: true,
+          state: {
+            backgroundLocation: { pathname: RouteUrls.Home },
+          },
+        });
       },
 
       toOperationRejectedStep(description?: string) {
         return navigate(RouteUrls.LedgerOperationRejected, {
           replace: true,
-          state: { description },
+          state: {
+            backgroundLocation: { pathname: RouteUrls.Home },
+            description,
+          },
         });
       },
 
       toDeviceDisconnectStep() {
-        return navigate(RouteUrls.LedgerDisconnected, { replace: true });
+        return navigate(RouteUrls.LedgerDisconnected, {
+          replace: true,
+          state: {
+            backgroundLocation: { pathname: RouteUrls.Home },
+          },
+        });
       },
 
       toBroadcastErrorStep(error: string) {
-        return navigate(RouteUrls.LedgerBroadcastError, { replace: true, state: { error } });
+        return navigate(RouteUrls.LedgerBroadcastError, {
+          replace: true,
+          state: {
+            backgroundLocation: { pathname: RouteUrls.Home },
+            error,
+          },
+        });
       },
 
       cancelLedgerAction() {
-        return navigate('..', { relative: 'path', replace: true, state: { ...location.state } });
+        // for send ordinal '..' brings you back to the `choose-fee` step but you lose context of the ordinal
+        const backLocation = location.pathname.match(RouteUrls.SendOrdinalInscription)
+          ? RouteUrls.Home
+          : '..';
```

Comment:
It's never too late to revert anything. And this is a real pain I agree. 

Unfortunately `..` seems to give un-predictable results and in this scenario with ledger it just sends them back to `send/ordinal-inscription/choose-fee` but doesn't keep the fee modal open.

Reply:
As far as I understand the problem, with `/*` anything that comes after is treated as a single route via wildcard match, thus breaking the ledger implementation that relies on route changes

Reply:
@pete-watters wonder if you can try something:

We're missing a path on the outer route. I wonder if that means it thinks `/*` routes are the same, but if we put a path in, does that behaviour change?

```
export function ledgerSignTxRoutes({ component, customRoutes }: LedgerSignTxRoutesProps) {
  return (
    <Route path="/" element={component}>
      {customRoutes}
```

Reply:
Good idea. I'll test that now and try see if I can both fix this and change the pattern to be a bit cleaner.

Reply:
@kyranjamie if I add this route wrapper the whole APP won't even load, crashing with: 
![Screenshot 2024-04-19 at 11 56 25](https://github.com/leather-wallet/extension/assets/2938440/818e88bf-9f9c-49ea-90cc-f24fdaee30cd)

I think it would be prudent to merge this fix for a P1 bug first, then continue efforts refactoring the routes to implement a more permanent solution for this and all of our routes. 

The routing setup now is also proving tricky when implementing the Approval UX so I think it's worth re-reviewing it as a hole.

I checked back and I originally added the `/*` to the route to suppress this warning: 
![Screenshot 2024-04-19 at 12 00 16](https://github.com/leather-wallet/extension/assets/2938440/a644f645-31bb-4666-8b8c-fd7be64f28e3)

I will take a look at the suggestions that provides to use `/` or `*` but I think that was causing some other issues. 

It could be better to remove the `/*` get rid of this and accept that we will get this warning but have a simpler setup